src/settings/JobProfiles/JobProfilesContainer/JobProfilesContainer.js

```diff
@@ -8,20 +8,15 @@ import {
 } from 'react-router-prop-types';
 
 import { Route } from '@folio/stripes/core';
-import { makeQueryFunction } from '@folio/stripes/smart-components';
 import {
   JobProfiles,
   getJobProfilesColumnProperties,
   DEFAULT_JOB_PROFILES_COLUMNS,
   getJobProfilesItemFormatter,
 } from '@folio/stripes-data-transfer-components';
 
+import { manifest } from '../../../components/ChooseJobProfile/manifest';
```

Comment:
This is not clear, why the manifest from JobProfilesContainer was moved to ChooseJobProfile and use it now. It should be at least the shared place. Because now the settings page depends on the component from the main page and every change to ChooseJobProfile might affect JobProfiles settings

Reply:
Let's decide on the place for that, I thought about this too. I would think then of the srs/common folder.